### PR TITLE
docs(release): make testing of new features explicit

### PR DESCRIPTION
**Pull request by marstamm**
closes https://github.com/bpmn-io/internal-docs/issues/378

### CODE REVIEW

File: docs/.project/INTEGRATION_TEST.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ __Target:__ Perform tests on nightly builds on supported platforms.
 
 * [ ] fetch [latest release/nightly](https://camunda.org/release/camunda-modeler/)
 * [ ] fetch [latest version of Cawemo plugin](https://downloads.camunda.cloud/enterprise-release/cawemo/cloud-connect-modeler-plugin/)
+* [ ] test changes of the release (see [release issue](https://github.com/camunda/camunda-modeler/labels/release) and [changelog](https://github.com/camunda/camunda-modeler/blob/develop/CHANGELOG.md))
```
Comment (MaxTru) [inline docs/.project/INTEGRATION_TEST.md:15]:
```suggestion
* [ ] test changes of the release from the users' perspective end-2-end (e.g., include deployment where appropriate) (see [release issue](https://github.com/camunda/camunda-modeler/labels/release) and [changelog](https://github.com/camunda/camunda-modeler/blob/develop/CHANGELOG.md))
* [ ] ```

**Review by MaxTru — CHANGES_REQUESTED** [review]
Good stuff! 

Just a minor change suggested

**Review by MaxTru — APPROVED** [review]

**Comment by nikku** [comment]
It looks like we've finally reached the maximum URL length.

New releases cannot longer easily be crafted: 

image

**Comment by MaxTru** [comment]
> It looks like we've finally reached the maximum URL length.
> 
> New releases cannot longer easily be crafted:
> 
> image

:hankey:

**Comment by MaxTru** [comment]
I opened https://github.com/camunda/camunda-modeler/issues/2549 as follow-up issue

**Pull request merged**